evaluation/pipeline_invoker.py

```
@@ -0,0 +1,126 @@
+"""In-process pipeline invocation for evaluation."""
+
+from __future__ import annotations
+
+import logging
+
+from core.config import BaseConfig
+from core.container import Container
+from core.events.input import Input
+from core.events.response import Source
```

Flake8 (7.3.0)

[error] 'core.events.response.Source' imported but unused

(F401)